src/bid_order.rs

```diff
@@ -115,27 +108,7 @@ impl BidOrderV2 {
     pub fn get_remaining_fee(&self) -> Uint128 {
         match &self.fee {
             None => Uint128::zero(),
-            Some(fee) => {
-                fee.amount
-                    - self
-                        .events
-                        .iter()
-                        .map(|event| match &event.action {
-                            Action::Fill { fee, .. } => match fee {
-                                None => Uint128::zero(),
-                                Some(fee) => fee.amount,
-                            },
-                            Action::Refund { fee, .. } => match fee {
-                                None => Uint128::zero(),
-                                Some(fee) => fee.amount,
-                            },
-                            Action::Reject { fee, .. } => match fee {
-                                None => Uint128::zero(),
-                                Some(fee) => fee.amount,
-                            },
-                        })
-                        .sum::<Uint128>()
-            }
+            Some(fee) => fee.amount - self.remaining_fee,
```

Comment:
would this just be `self.remaining_fee`?

Reply:
it's `self.base.amount - self.accumulated_fee` now